actions/setup_e2e_tests.sh

```diff
@@ -86,9 +84,7 @@ elif [[ -n "$DEBTEST" ]]; then
     fi
 
     # Install from GitHub
-    # Ubuntu 16.04 has both bats and jq packages, so we don't need to do this
-    # once we drop Ubuntu 14.04 support
-    git clone --branch add_per_test_timing_information --depth 1 https://github.com/Kami/bats-core.git
+    git clone https://github.com/bats-core/bats-core.git
```

Comment:
Is the bats 0.4.0 deb too old? The previous comment suggests that we won't need the repo once we move off 14.04.

Reply:
To be honest, I'm not 100% sure. I figured we could try out that change in a future PR - I just wanted to make the quickest change to remove Ubuntu 14 support. I'll create an issue to track this separately.

Reply:
Tracked in https://github.com/StackStorm/st2cd/issues/434 - resolving.